test/User-test.js

```diff
@@ -0,0 +1,48 @@
+const chai = require('chai');
+const expect = chai.expect;
+const sampleData = require('../test/sampleData');
+// const userData = sampleData[0];
+const User = require('../src/User');
+
+
+describe('User', function() {
+  let user, example;
+
+  beforeEach(function() {
+    userData = {"id": 1,
+      "name": "Luisa Hane",
+      "address": "15195 Nakia Tunnel, Erdmanport VA 19901-1697",
+      "email": "Diana.Hayes1@hotmail.com",
+      "strideLength": 4.3,
+      "dailyStepGoal": 10000,
+      "friends": [
+        16,
+        4,
+        8
+      ]
+    };
+    user = new User(userData);
+  });
+
+  it('should be a function', function() {
+    expect(User).to.be.a('function');
+  });
+
+  it('should instantiate a User', function() {
+    expect(user).to.be.an.instanceof(User);
+  });
+
+  it('should hold all the user data', function() {
```

Comment:
**suggestions**:

1. If you get the `userData` object itself in your test, you can directly compare the properties:
```
    expect(user.name).to.equal(expectedUser.name);
```
2. Once you do that, you can easily add test data by making your `userData` into an array of objects. Testing a variety of user data is as easy as using a for-loop. (For example, empty properties, properties with null values, etc.)